vtigercron.php

```diff
@@ -22,12 +22,16 @@
 		// Run specific service
 		$srv = empty($argv[1]) ? $_REQUEST['service'] : $argv[1];
 		$srv = vtlib_purify($srv);
-		$srvcron = Vtiger_Cron::getInstance($srv);
-		if ($srvcron !== false) {
-			$cronTasks = array($srvcron);
+		if ($srv == '--quiet' || $srv == '-q') {
+			// do nothing
```

Comment:
I think you have to load all cron tasks here: `$cronTasks = Vtiger_Cron::listAllActiveInstances();` 
When we use the -q option we want to execute all the tasks without any messages, not ignore them all.